<launch>

    <!-- 启动 TEB 触发区域和目标航点 可视化节点 -->
    <node pkg="f1" type="rviz_teb_area_goal.py" name="teb_zone_standalone_visualizer" output="screen" /> 

    <!-- FTG 可视化节点 -->
    <node pkg="f1" type="rviz_ftg.py" name="ftg_visualizer_node" output="screen" />  

    <!-- RQT 动态调参节点 -->
    <include file="$(find f1)/launch/include/RQT.launch" />

    
</launch>